### PR TITLE
login-page-added

### PR DESCRIPTION
**Pull request by Jia-A**
Screenshot (461)
-- This is the Login page for user so that he can access the wishlist and cart pages and also see the recommended products according to his previous searching and buying.
-- Both the fields in the login form are necessary.

### CODE REVIEW

File: site-pages/login/login.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+.h4{
+    font-weight: 300;
+}
+.new-ac{
+    text-decoration: underline;
+}
+.submit-btn{
+    width: 85%;
+}
```
Comment (dev-enforced) [inline site-pages/login/login.css]: Saw this class in the signup.css file so you can place it in a common file of your choice and use it in the places where you intend to

File: site-pages/login/login.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+.label-inp{
+
+    margin-bottom: 1px;
+}
+.login-head{
+    font-weight: 500;
+    color: var(--primary-color);
+
+}
```
Comment (dev-enforced) [inline site-pages/login/login.css]: Saw these classes while developing the signup css file https://github.com/Jia-A/dot-store/pull/6/files so if both of these css files(login.css and signup.css) are similar then you can create a common css file named as auth.css and store these common styles within it and keep the remaining things in these two files

**Review by dev-enforced — APPROVED** [review]
The code looks good to me !!Do tryout the suggestions mentioned below and then you can merge it

File: site-pages/login/login.css (hunk 1 above)
Comment (Jia-A) [reply site-pages/login/login.css]: Okay, I will put it in utility class and then use it.

File: site-pages/login/login.css (hunk 2 above)
Comment (Jia-A) [reply site-pages/login/login.css]: Okay, will do that definitely.

File: site-pages/login/login.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
+.login-main{
+    display: flex;
+    margin-top: 30px;
```
Comment (saadahmadhere) [inline site-pages/login/login.css]: relative units would be better.

File: site-pages/login/login.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    <link rel="stylesheet" href="login.css">
+
+    <title>Dot | Login</title>
+</head>
+<body>
+    <div class="nav-con">
+        <nav class="nav-bar">
+            <div>
+                <h2 class="website-name">Dot Store</h2>
+
```
Comment (saadahmadhere) [inline site-pages/login/login.html:22]: Space is not required here in my opinion.

**Review by saadahmadhere — APPROVED** [review]
Code looks nice and well written. It's good to merge.

**Pull request merged**